### PR TITLE
Add monitoring to the sample setup

### PR DESCRIPTION
**Pull request by nkoester**
This PR implements the monitoring descriptions recently added to the `model-server`.

### CODE REVIEW

File: docs/global/modules/samples/pages/monitoring.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+----
+
+This command starts the following components:
+
+* `model-server` at http://localhost:28101/[],
+* Prometheus database at http://localhost:9090/[], and
+* Grafana dashboard at http://localhost:3001/[].
+
+
+NOTE: Make sure you picked the correct instance at the top of the Grafana dashboard.
```
Comment (languitar) [inline docs/global/modules/samples/pages/monitoring.adoc:24]: What makes the instance "correct". Could be a bit more specific.

File: docs/global/modules/samples/pages/monitoring.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+[NOTE]
+====
+Depending on your operating system or docker setup, you might need to adjust the targets defined in the `prometheus.yml` and `datasource.yml`.
+
+Also, the docker network setup might need adjustments in the `docker-compose.yaml`.
```
Comment (languitar) [inline docs/global/modules/samples/pages/monitoring.adoc:30]: Which adjustments and why? Apart from conflicting port-forwards with other services on the host, I don't see what could need changes?

File: docs/global/modules/samples/pages/monitoring.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+====
+
+
+=== Engage with the `model-server`
```
Comment (languitar) [inline docs/global/modules/samples/pages/monitoring.adoc:34]: sounds romantic ;-)

**Review by languitar — APPROVED** [review]
Some minor clarification requests and ideas inside. Generally fine.

File: docker-compose.yaml [hunk 4]
```diff
@@ -102,3 +103,39 @@ services:
...
+    restart: unless-stopped
+    volumes:
+      - ./monitoring/prometheus:/etc/prometheus
+      - prom_data:/prometheus
+
+  grafana:
+    image: grafana/grafana-oss
+    container_name: modelix-samples-grafana
+    ports:
+      - 3001:3000
```
[the remark this answers is not in the document]
Comment (nkoester) [reply docker-compose.yaml:126]: 3000 is already blocked by other SPAs in the project

File: docs/global/modules/samples/pages/monitoring.adoc (hunk 3 above)
Comment (nkoester) [reply docs/global/modules/samples/pages/monitoring.adoc:34]: We all should consider it

**Pull request merged**